roles/collectd/tasks/main.yml

```diff
@@ -1,12 +1,11 @@
 ---
-- name: install collectd
-  yum:
-      name: '{{ item }}'
-      state: present
-  with_items:
-      - collectd
```

Comment:
Inspired from https://github.com/centos-opstools/opstools-ansible/blob/master/roles/fluentd/server/tasks/main.yml#L2-L8. But I can sure make it obvious. Will update.